### ISSUE TITLE
SCRPTSAVE.SAV is left empty

### ISSUE THREAD

**Issue by WalterStieben** [elsky42/ninja-bonus_independent_training#4]
A few hours ago, Spine has updates this mod from version v0.3.X to v0.7.3. This current version introduces a bug that affects saving and loading. Apperently, this mod causes the "SCRPTSAVE.SAV" file in a save folder to be empty. An attempt to load a save slot that was damaged in this way causes the game client to crash with multiple error messages.

There are ways to temporarily mitigate the problem:
- Start the game client and load a save file with an intact SCRIPTSAVE.SAV file. After that it is possible to load other save files until the game client is closed.
- Manually replace an empty SCRIPTSAVE.SAV file with an intact one. The corresponding save file can be loaded until the save slot is overwritten.

Of course, disabling the mod is a possible solution as well, but we want to keep using it, right?

Additional information about the game version or similar details can be provided if necessary.

**Comment by elsky42** [comment]
I'm not sure what can cause this issue, I've never run into it and I finished the game with the mod enabled. Can you provide me the version of Gothic 2 and all the mods you installed?

**Comment by WalterStieben** [comment]
Okay here are all the details that I can think of. Hope it helps!

Software:
- Spine 1.43.0
- Gothic II: Night of the Raven 2.6.1

Other mods:
- 4GB Patch 1.0.0
- AutoRun  1.0.0
- Categorized Inventory 1.0.1
- Infinite Sprint 2.3.1
- Item Locator 1.0.0
- Item Map 1.6.0
- Ninja 2.7.12
- OrderedLogs 1.0.2
- Patch Menu 1.0.0
- PickLockHelper 1.6.1
- QuickOpen 1.1.1
- Quickloot 1.9.5
- QuicksaveSlots 1.0.1
- RenderBarValues 1.0.0
- Show Game Time 1.0.7
- Systempack 1.8.2
- Texturepack by Artemiano 1.0.5

So this is a mix that includes mostly QoL mods. Some of them are Ninja mods as well.

Just to document my whole experience so far:
- I was playing a save with this mod enabled from the beginning. The version should have been either 0.3.0 or 0.3.1.
- This mod was updated on Spine to the new version 0.7.3.
- New save files now had the empty SCRIPTSAVE.SAV file as described above.
- Disabling this mod in Spine and manually downgrading to version 0.3.1 resolved the problem so far.
- All other mods work together flawlessly and the same is true for the older version of this mod.

I have done some further testing:
- The problem reappears when the Spine version of this mod is turned on again and disappears when it is turned off.
- Freshly starting a new game does not change this behavior.
- Deactivating all other mods (aside from this mod, Ninja and Systempack) DOES (!) help.
- Exhaustively testing all combinations of other mods with this mod I was able to divide my mod list into mods that are compatible with the current version of this mod and those which are not.
- The incompatible ones are:
  - QuickOpen
  - Show Game Time
  - RenderBarValues
  - Quickloot
  - Picklockhelper
  - OrderedLogs
  - Infinite Sprint
  - AutoRun
- All incompatible mods are Ninja mods but there are some Ninja mods which are compatible.
- All in all, the testing strongly suggests that the problem originates from this mod.

**Comment by B2SquareOne** [comment]
I have the same issue.
The past week I tried so much, installed gothic multiple times, installing Mods one by one and it's always the same outcome. When I install your mod I get the crash as described by WalterStieben.

Can you please help? I like your mod so much and really want a fresh playthrough with this mod.

**Comment by elsky42** [comment]
I'm doing some testing but it may take some time to figure out what the issue is.

**Comment by B2SquareOne** [comment]
I very much appreciate it and your work! Thank you

**Comment by B2SquareOne** [comment]
If I can help you in any way shape or form please tell me. Or if I should test anything..

**Comment by elsky42** [comment]
I've been able to reproduce the issue and it should be solved now. Can you try 0.7.4 and tell me if it works?

**Comment by B2SquareOne** [comment]
of course I will try!

**Comment by B2SquareOne** [comment]
I tested it and the error (i guess around SCRIPTSAVE.SAV file) is gone! Restartet multiple time and no crash! 

But I have been playing around for a few minutes and found something new:
My Char has STR 10 / 11 (I prayed for 100 Gold for +1 Strengh)
If I train now + 5 Str I get the following = 20 / 16

Seems like the first number(20) gets doubled as my new false base amount of that attribute. And the second (16) should be the right new base amount but is falsely shown as the base+boni attribute. You know what I mean
